### ISSUE TITLE
Possibly Incorrect Sources

### ISSUE THREAD

**Issue by dssjon** [HistoricalChristianFaith/Commentaries-Database#8]
The following example reflects a quote attributed to Aquinas yet the .toml file is located within the Origen of Alexandria folder. Is this an incorrectly located file or am I misunderstanding the project structure? Example:

https://github.com/HistoricalChristianFaith/Commentaries-Database/blob/master/Origen%20of%20Alexandria/Matthew%2024_6-8.toml#L12

Thanks

**Comment by emeth-** [comment]
So in this case, Aquinas is quoting Origen! 

Generally, we try to list primary sources (e.g. if Origen says X in a commentary on Matthew, we insert that quote under "Origen of Alexandria" with the source pointing to his commentary on Matthew).

But sometimes the original source is lost. For example, Papias' original works are all lost, but [his commentary on Isaiah 11:6-7](https://github.com/HistoricalChristianFaith/Commentaries-Database/blob/master/Papias%20of%20Hierapolis/Isaiah%2011_6-7.toml) is quoted by Irenaeus in his Against Heresies around 202 AD. In that case, we store the quote under "Papias of Hierapolis", but add an additional tag of `append_to_author_name=' (as quoted by Irenaeus, AD 202)'`, and point the source to Irenaeus' work.

Note that there are a few other cases, since the database is a continual work in progress: 
- We loaded all of Aquinas' Catena Aurea, since that was a simple way to seed the database with many quotations from earlier church fathers. Many (but not nearly all) of those we have now loaded original sources for as well. So these are examples where original sources exist, but we just haven't linked them up yet and so we're currently relying on Aquinas' quotations of them.
- In some databases we loaded, we have a quotation attributed to a church father, but no source! We've been working hard on attributing these properly to direct sources, but it's still a work in progress. An example of this is [Dionysius of Corinth on Matthew 27:50](https://github.com/HistoricalChristianFaith/Commentaries-Database/blob/master/Dionysius%20of%20Corinth/Matthew%2027_50.toml). In these cases, we decided to err on the side of including too much information rather than excluding less then perfect information (as people can filter out unattributed quotations if they desire when loading them into a database).
